packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -263,6 +272,30 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
     }
   }
 
+  /**
+   * Returns the required ratio of yes:no votes needed to exceed in order to pass the proposal.
+   * @param tx Transaction to determine the constitution for running.
+   */
+  async getConstitution(tx: ProposalTransaction): Promise<BigNumber.Value> {
```

Comment:
can we make a utility for a tx array/`Proposal` type like the CLI uses?
@mcortesi is a big advocate of a "dumb" CLI and a "featureful" kit

Reply:
Renamed this function and changed this signature to take a proposal.